src/parser/jobs/smn/modules/MultiHitSkills.js

```diff
@@ -1,65 +1,37 @@
-import {t} from '@lingui/macro'
 import ACTIONS from 'data/ACTIONS'
-import Module from 'parser/core/Module'
-import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
-import {Plural, Trans} from '@lingui/react'
-import {ActionLink} from 'components/ui/DbLink'
-import React from 'react'
+import {AoEUsages} from 'parser/core/modules/AoEUsages'
 
-const REQUIRED_MULTI_HIT_TARGETS = {
-	[ACTIONS.PAINFLARE.id]: 2,
-	[ACTIONS.ENERGY_SIPHON.id]: 3,
-	[ACTIONS.OUTBURST.id]: 3,
-}
-
-const SEVERITY_LOW_TARGET_ATTACKS = {
-	1: SEVERITY.MINOR,
-	2: SEVERITY.MEDIUM,
-	4: SEVERITY.MAJOR,
-}
-
-export default class MultiHit extends Module {
-	static handle = 'multihit'
-	static title = t('smn.multihit.title')`Multi-Hit`
+export default class MultiHit extends AoEUsages {
 	static dependencies = [
 		'gauge',
-		'suggestions',
+		//need to include base class dependency or it will not be pulled in
```

Comment:
This is brittle, relies on anyone changing the base module to check that this is also updated.

Using old-style js modules like this, it's usually handled by spreading the parent's dependencies in:

```js
static dependencies = [
	...ParentModule.dependencies,
	'newDependency',
]
```

Having said that, I'm not _entirely_ sure if that will work when the base class is using `@dependency` - a quick check should make it clear. If it _doesn't_ work, adding `@dependency gauge` should work fine in JS, and will work.

Reply:
I was not aware of this syntax for this, I'll change it in the next go.